pom.xml

```diff
@@ -54,4 +59,21 @@
         <slf4jVersion>1.7.32</slf4jVersion>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+                <executions>
```

Comment:
I'm getting a warning in IDEA about missing version, but good to see this here 👍